Extensions/accesskit.js

```diff
@@ -131,6 +136,8 @@ XKit.extensions.accesskit = new Object({
 
 		XKit.tools.init_css('accesskit');
 
+		$.fx.off = this.preferences.xkit_disable_animation.value ? true : false;
```

Comment:
since `preferences.xkit_disable_animation.value` is already a boolean value, couldn't this just be

```suggestion
		$.fx.off = this.preferences.xkit_disable_animation.value;
```
?

Reply:
whoopsie! yep, left in from earlier code toggling it where it defaults to undefined